web/cobrands/northamptonshire/assets.js

```diff
@@ -109,59 +87,13 @@ var layers = [
   "categories": [ "Sign/Waymarking - Damaged/Missing" ],
   "item_name": "waymarking or right of way",
   "layer_name": "WAYMARK POST",
-  "styleid": "5d481a4ffe2ad809d85a7b90"
-},
-/*
-{
-  "categories": [
-    "Damaged/Exposed Wiring / Vandalised",
-    "Lamp/Bulb Failure",
-    "Signal Failure",
-    "Signal Failure all out",
-    "Signal Stuck",
-    "Signal Head Failure",
-    "Request Timing Review",
-    "Damaged Control box",
-    "Signal Failure/Damaged - Toucan/Pelican"
-  ],
-  "item_name": "signal or crossing",
-  "layer_name": "Midblock",
-  "layer": 223,
-},
-{
-  "categories": [
-    "Damaged/Exposed Wiring / Vandalised",
-    "Lamp/Bulb Failure",
-    "Signal Failure",
-    "Signal Failure all out",
-    "Signal Stuck",
-    "Signal Head Failure",
-    "Request Timing Review",
-    "Damaged Control box",
-    "Signal Failure/Damaged - Toucan/Pelican"
-  ],
-  "item_name": "signal or crossing",
-  "layer_name": "TL Junction",
-  "layer": 225,
-},
-*/
-{
-  "categories": [
-      "Fallen Tree",
-      "Restricted Visibility / Overgrown / Overhanging",
-      "Restricted Visibility"
-  ],
-  "layer_name": "Tree",
-  "styleid": '5d481376fe2ad8059ce44ef2',
-},
-{
-  "categories": [ "Safety Bollard - Damaged/Missing" ],
-  "layer_name": "Safety Bollard",
-  "styleid": "5d481446fe2ad8059ce44f02",
+  "styleid": "5d481a4ffe2ad809d85a7b90&styleIds=5d481742fe2ad809d85a7b05"
```

Comment:
Is this adding another layer also?

Reply:
This is fetching two styles in the same call and displaying them on the same layer. In Alloy the assets are grouped into layers in which individual asset types are styles and you can request multiple styles in one call.